gulpfile.js

```diff
@@ -176,9 +175,9 @@ gulp.task('test', ['testResources'], function () {
     shell.mkdir('-p', _testTemp);
 
     var suitePath = path.join(_testRoot, options.suite + '/_suite.js');
-
+    var tfBuild = ('' + process.env['TF_BUILD']).toLowerCase() == 'true'
```

Comment:
The CI flag doesn't *need* to be removed, I just thought it might be neat to leverage our TF_BUILD variable instead :)